code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -643,3 +643,9 @@
 		var/custom_data = item.on_object_saved(depth++)
 		dat += "[custom_data ? ",\n[custom_data]" : ""]"
 	return dat
+
+/obj/structure/closet/proc/on_magic_unlock(datum/source, datum/action/cooldown/spell/aoe/knock/spell, mob/living/caster)
```

Comment:
Where is the signal for this being registered at?

Reply:
Now it's registered, terribly sorry